nixos/modules/module-list.nix

```diff
@@ -219,12 +219,13 @@
   ./programs/kbdlight.nix
   ./programs/kclock.nix
   ./programs/kdeconnect.nix
+  ./programs/kubeswitch.nix
   ./programs/ladybird.nix
   ./programs/lazygit.nix
-  ./programs/kubeswitch.nix
   ./programs/less.nix
```

Comment:
Could you leave this sort in another commit, or for another PR? It's not really relevant for localsend.

Reply:
The sort has been reverted.